services/user_manager.js

```diff
@@ -36,4 +36,27 @@ exports.process_update_user_by_id = async (input_data, req) => {
 
 exports.post_process_update_user_by_id = async (resp) => {
     resp.status(200).send({status:"success",message:"user updated successfully"})
+}
+
+
+exports.pre_process_get_user_enrollments = async (req) => {
+    console.log("user_id",req.params.id)
+    return req.params.id
+}
+
+exports.process_get_user_enrollments = async (user_id) => {
+    console.log("user_id 1",user_id)
+    const enrollments = await models.Enrollment.findAll({
+        where: {
+            user_id: user_id,
+            status:"active"
```

Comment:
in UI there can be a section to display user's previous enrollements.we should handle the status field either through params or for now we can pass all the enrollments of any user and filteration of active can be done on UI.discuss with arihant if needs only active enrollments

Reply:
right